core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -151,13 +141,23 @@ class InvokerPool(
     }
 
     /** Receive Ping messages from invokers. */
-    pingConsumer.onMessage((topic, _, _, bytes) => {
+    val pingPollDuration = 1.second
+    val invokerPingFeed = context.system.actorOf(Props {
+        new MessageFeed("ping", logging, pingConsumer, pingConsumer.maxPeek, pingPollDuration, processInvokerPing, logHandoff = false)
+    })
+
+    def processInvokerPing(bytes: Array[Byte]): Future[Unit] = Future {
         val raw = new String(bytes, StandardCharsets.UTF_8)
         PingMessage.parse(raw) match {
-            case Success(p: PingMessage) => self ! p
-            case Failure(t)              => logging.error(this, s"failed processing message: $raw with $t")
+            case Success(p: PingMessage) =>
+                self ! p
+                invokerPingFeed ! MessageFeed.Processed
+
+            case Failure(t) =>
+                invokerPingFeed ! MessageFeed.Processed
```

Comment:
i duped so that the message goes out before the logger but it's probably inconsequential in the totality.